atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/backend/CassandraKeyValueServiceInstrumentation.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2016 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.performance.backend;
+
+
+import java.net.InetSocketAddress;
+
+import com.google.common.base.Optional;
+import com.google.common.collect.ImmutableSet;
+import com.palantir.atlasdb.cassandra.CassandraKeyValueServiceConfig;
+import com.palantir.atlasdb.cassandra.CassandraKeyValueServiceConfigManager;
+import com.palantir.atlasdb.cassandra.ImmutableCassandraCredentialsConfig;
+import com.palantir.atlasdb.cassandra.ImmutableCassandraKeyValueServiceConfig;
+import com.palantir.atlasdb.config.ImmutableLeaderConfig;
+import com.palantir.atlasdb.keyvalue.cassandra.CassandraKeyValueService;
+import com.palantir.atlasdb.spi.KeyValueServiceConfig;
+
+public class CassandraKeyValueServiceInstrumentation extends KeyValueServiceTypeInterface {
+
+    public CassandraKeyValueServiceInstrumentation() {
+        super(9160, "cassandra-docker-compose.yml");
+    }
+
+    @Override
+    public KeyValueServiceConfig getKeyValueServiceConfig(InetSocketAddress addr) {
+        return ImmutableCassandraKeyValueServiceConfig.builder()
+                .addServers(addr)
+                .poolSize(20)
+                .keyspace("atlasdb")
+                .credentials(ImmutableCassandraCredentialsConfig.builder()
+                        .username("cassandra")
+                        .password("cassandra")
+                        .build())
+                .ssl(false)
+                .replicationFactor(1)
+                .mutationBatchCount(10000)
+                .mutationBatchSizeBytes(10000000)
+                .fetchBatchCount(1000)
+                .safetyDisabled(false)
+                .autoRefreshNodes(false)
+                .build();
+    }
+
+    @Override
+    public boolean canConnect(InetSocketAddress addr) {
+        try {
+            CassandraKeyValueService.create(
+                    CassandraKeyValueServiceConfigManager.createSimpleManager(
+                            (CassandraKeyValueServiceConfig) getKeyValueServiceConfig(addr)),
+                    Optional.of(ImmutableLeaderConfig
+                            .builder()
```

Comment:
this is inconsistent with line 39 above - I'd lean towards having `.builder` on the same line.